.buildkite/docker-build-push-explorerApi.nix

```diff
@@ -55,8 +55,13 @@ in
       tag="''${BUILDKITE_TAG:-}"
       tagged="$fullrepo:$tag"
       gitrev="${image.imageTag}"
+      echo "Images before loading"
```

Comment:
Remove these debugging commands. Call `docker system prune -f` at the end of the script to clean up